config/routes.js

```diff
@@ -32,10 +32,7 @@ module.exports.routes = {
   *                                                                          *
   ***************************************************************************/
 
-  '/*': function log (req, res, next) {
-    sails.log.info(new Date(), req.method, req.url);
-    next();
-  },
+  '/*': { policy: 'angularRouting', skipAssets: true },
```

Comment:
@jamesjwarren I've tried but can't get it to work, or the `skipAssets` option. Will look at it some more.